### DISCUSSION TITLE
How to minify JS and ignore errors

### DISCUSSION THREAD

**Discussion by polycode-nz** [bdkjones/CodeKit#700] [Q&A] [answered]
I have a need to minify JS but ignore any errors it may kick up. Risky, sure.

Is that possible?

I set "Check syntax with" to nothing but no joy, still picks up the errors. I started looking through the Eslint settings - daunting.

To give some reasoning without too much derailing detail: I need to use some Laravel @can directives around some JS built buttons for a [datatables](https://datatables.net/) table.

The pipeline is the JS gets squished into a blade.php file and embedded/included into the Laravel view. So those @can directives are parsed out at runtime.

**Comment by bdkjones** [comment, marked as answer]
No, that isn't possible. If the minifier is throwing errors, that means it can't correctly parse the JS you're trying to minify, which means it can't create minified JS. There is no way to take JS-with-errors and turn it into minified-JS-with-errors.

**Reply by polycode-nz** [reply]
No problem, I figured. 

Thanks for replying
